src/window_help.cpp

```diff
@@ -60,3 +60,23 @@ void Window_Help::Clear() {
 	this->align = Text::AlignLeft;
 	contents->Clear();
 }
+
+void Window_Help::AddText(int x, std::string text, int color, Text::Alignment align, bool halfwidthspace) {
+	std::string::size_type pos = 0;
+	std::string::size_type nextpos = 0;
+	while (nextpos != std::string::npos) {
+		nextpos = text.find(' ', pos);
+		auto segment = ToStringView(text).substr(pos, nextpos - pos);
+		contents->TextDraw(x, 2, color, segment, align);
+		x += Font::Default()->GetSize(segment).width;
+
+		if (nextpos != decltype(text)::npos) {
+			if (halfwidthspace) {
+				x += Font::Default()->GetSize(" ").width / 2;
+			} else {
+				x += Font::Default()->GetSize(" ").width;
+			}
+			pos = nextpos + 1;
+		}
```

Comment:
Because this is a copy-paste from "SetText" you can make SetText call AddText, then the code is not duplciated anymore.

Reply:
Fixed.